lazyscribe/artifacts/json.py

```diff
@@ -0,0 +1,57 @@
+"""Artifact handler for JSON-serializable objects."""
+
+from json import dump, load
+from typing import ClassVar
+
+from attrs import define
+
+from .base import Artifact
+
+
+@define(auto_attribs=True)
+class JSONArtifact(Artifact):
+    """Handler for JSON-serializable objects."""
+
+    alias: ClassVar[str] = "json"
+    binary: ClassVar[bool] = False
+
+    @classmethod
+    def construct(cls):
+        """Construct the handler class.
+
+        Included for compatibility.
+        """
+        return cls()
```

Comment:
Might be overkill, but should we add Python version here in lieu of package version? Not sure if there have been breaking changes in the past with this pre packaged module (thinking along the lines of `yaml.load` requiring a `Loader` spec or use of `safe_load`), but it could prove helpful.